composables/transaction/evm/useMetaTransaction.ts

```diff
@@ -0,0 +1,118 @@
+import { type Address, TransactionExecutionError } from 'viem'
+import useTransactionStatus from '@/composables/useTransactionStatus'
+
+type EvmHowAboutToExecuteEvents = {
+  onSuccess?: (param: EvmHowAboutToExecuteOnSuccessParam) => void
+  onError?: () => void
+}
+
+export type EvmHowAboutToExecuteParam = {
+  account: Address
+  address: Address
+  functionName: string
+  abi: any
```

Comment:
nit: use unknown if possible instead of any

Reply:
done, noted